### PR TITLE
Thread prio

### PR DESCRIPTION
**Pull request by akallabeth**

### CODE REVIEW

**Comment by akallabeth** [comment]
@freerdp-bot test please

File: winpr/libwinpr/thread/thread.c [hunk 1]
```diff
@@ -605,6 +614,54 @@ static BOOL winpr_StartThread(WINPR_THREAD* thread)
 	return rc;
 }
 
+BOOL SetThreadPriority(HANDLE hThread, int nPriority)
+{
+	ULONG Type = 0;
+	WINPR_HANDLE* Object = NULL;
+
+	if (!winpr_Handle_GetInfo(hThread, &Type, &Object))
```
Comment (hardening) [inline winpr/libwinpr/thread/thread.c:622]: here we should also check that the object is a thread

File: winpr/libwinpr/thread/thread.c [hunk 2]
```diff
@@ -28,6 +28,14 @@
 
 #include <winpr/thread.h>
 
+#ifndef MIN
```
Comment (hardening) [inline winpr/libwinpr/thread/thread.c:31]: not your fault, but we 13 definitions of `MIN` in FreeRDP and 5 for `MAX`...

Comment (akallabeth) [reply winpr/libwinpr/thread/thread.c:31]: hmm, we already got a central definition for `freerdp` but lacking one for `winpr`

**Comment by akallabeth** [comment]
@freerdp-bot test again please

**Pull request merged**